frontend/app/components/transportation/study-area-map/census-tract-popup/modal-split.js

```diff
@@ -0,0 +1,36 @@
+import Component from '@ember/component';
+import { computed } from '@ember-decorators/object';
+
+/**
+ * ModalSplit component is responsible for correctly ingesting census-tract records
+ * and formatting the data for display. Due to the data model, multiple records are required
+ * to get the full data for a single census tract, and must be rendered correctly depending on
+ * which variable a single record is for. For variable = "population", the data is presented separately.
+ * All other variables are displayed in a table (see template for structure)
+ */
+export default class TransportationStudyAreaMapCensusTractPopupModalSplitComponent extends Component {
+  /**
+   * RecordArray from census-tract-popup/data containing census-tract models for a given geoid
+   */
+  data = [];
+
+  /**
+   *  The 'value' property of the census-tract record for variable = 'population'
+   */
+  @computed('data')
+  get population() {
+    const data = this.get('data');
+    const populationVariable = data.filter((censusTractVariable) => {return censusTractVariable.variable === 'population'})[0];
```

Comment:
Nit: Destructuring looks nicer: 
```suggestion
    const [populationVariable] = data.filter((censusTractVariable) => {return censusTractVariable.variable === 'population'});
```

TODO: enable better linting rules